src/tools/perf/lib/ucp_tests.cc

```diff
@@ -725,6 +725,39 @@ class ucp_perf_test_runner {
         wait_send_window(m_max_outstanding);
     }
 
+    bool use_ack() const
+    {
+        /* TODO: daemon does not support bi-directional flow */
+        if (m_perf.params.ucp.is_daemon_mode) {
```

Comment:
I am assuming this boolean is always valid on both ends of the perftest in daemon mode, is that correct?

Reply:
yes, it's valid on both ends. In general the test run config is shared between sender and receiver